docs/version0.15.md

```diff
@@ -32,6 +32,7 @@
 - [Support for pause-less garbage collection on IBM Z systems](#support-for-pause-less-garbage-collection-on-ibm-z-systems)
 - ![Start of content that applies only to Java 11 (LTS)](cr/java11.png) [ChaCha20 algorithm support for OpenSSL](#chacha20-algorithm-support-for-openssl)![End of content that applies only to Java 11 (LTS)](cr/java_close_lts.png)
 - [Support for OpenJDK HotSpot options](#support-for-openjdk-hotspot-options)
+- [Support for automatic huge pages allocation](#support-for-automatic-huge-pages-allocation)
```

Comment:
not sure if the use of words `automatic allocation` is ideal, automatic hugepage allocation is already supported by OpenJ9 through portlibrary using `HugeTlb` which uses system hugepages if possible

transparent hugepage with madvise is a promotion flag where the memory region marked will be promoted to hugepage by kernel if possible.

Maybe *`Support for Transparent HugePage`* would be better? (not sure on this)

Reply:
I've changed the title and references as you suggest. Please take a quick look and make sure you are happy before we merge. Thanks!